docs/workflow/README.md

```diff
@@ -12,81 +18,72 @@ The repo can be built for the following platforms, using the provided setup and
 | ARM64 | &#x2714; | &#x2714; | &#x2714; |          |
 |       | [Requirements](requirements/windows-requirements.md) | [Requirements](requirements/linux-requirements.md) | [Requirements](requirements/macos-requirements.md) | [Requirements](requirements/freebsd-requirements.md)
 
-Before proceeding further, please click on the link above that matches your machine and ensure you have installed all the prerequisites for the build to work.
-
 Additionally, keep in mind that cloning the full history of this repo takes roughly 400-500 MB of network transfer, inflating to a repository that can consume somewhere between 1 to 1.5 GB. A build of the repo can take somewhere between 10 and 20 GB of space for a single OS and Platform configuration depending on the portions of the product built. This might increase over time, so consider this to be a minimum bar for working with this codebase.
 
-## Concepts
-
-The runtime repo can be built from a regular, non-administrator command prompt, from the root of the repo, as follows:
-
-For Linux and macOS
-```bash
-./build.sh
-```
+## Getting Yourself Started
 
-For Windows:
-```cmd
-build.cmd
-```
+The runtime repo can be built from a regular, non-administrator command prompt, from the root of the repo.
 
-This builds the product (in the default debug configuration), but not the tests.
+The repository currently consists of three different major parts:
```

Comment:
wouldn't it make sense to separate coreclr and mono here? the'yre in their own trees, have their own build subsets etc.

Reply:
eh, it's clear below